modin/config/envvars.py

```diff
@@ -123,7 +123,7 @@ class RayRedisAddress(EnvironmentVariable, type=ExactStr):
 
 class CpuCount(EnvironmentVariable, type=int):
     """
-    How may CPU cores to utilize across the whole distribution
+    How many CPU cores to use when initialization of the Modin engine.
```

Comment:
`How many CPU cores to use during initialization of the Modin engine.` sounds better.

Reply:
Ok, lets leave your version.

Reply:
CPU cores are used not only at the time of initialization, but also in the process of all modin calculations.

It would also be nice to add a note on how to understand this variable in cluster computing.